Libplanet/Blocks/Block.cs

```diff
@@ -41,6 +42,10 @@ public class Block<T> : ISerializable
             Timestamp = timestamp;
             Transactions = transactions.ToArray();
             Hash = Hashcash.Hash(ToBencodex(false, false));
+            OrderedTransactions =
+                Transactions
+                    .OrderBy(tx =>
+                        new BigInteger(tx.Id.ToByteArray()) ^ new BigInteger(Hash.ToByteArray()));
```

Comment:
I think that materializing here(using `.ToList()` or `.ToArray()`) is better than now to avoid unexpected re-calculation.